components/src/components/elements/pricing-table/Plan.tsx

```diff
@@ -170,7 +154,9 @@ export const Plan = ({
                 ? t("Usage-based")
                 : isFreePlan && showZeroPriceAsFree
                   ? t("Free")
-                  : formatCurrency(planPrice ?? 0, planCurrency)}
+                  : showAsMonthlyPrices && selectedPeriod === PriceInterval.Year
+                    ? formatCurrency((planPrice ?? 0) / 12, planCurrency)
+                    : formatCurrency(planPrice ?? 0, planCurrency)}
             {!plan.custom && !isFreePlan && <sub>/{selectedPeriod}</sub>}
```

Comment:
Should the copy also be changed here? When showAsMonthlyPrices is true and selectedPeriod === "year", this still displays "/year" but the price is monthly.